skrf/calibration/deembedding.py

```diff
@@ -908,4 +912,834 @@ def deembed(self, ntwk):
         h_ = h.flipped()
         caled.z = (h.z + h_.z) / 2
 
-        return caled
\ No newline at end of file
+        return caled
+
+
+class Ieeep370nzc2xthru(Deembedding):
+    """
+    IEEEP3702xThru.m creates error boxes from a test fixture 2x thru.
+    
+    Based on https://opensource.ieee.org/elec-char/ieee-370/-/blob/master/TG1/IEEEP3702xThru.m
+    commit 49ddd78cf68ad5a7c0aaa57a73415075b5178aa6
+    See github issue https://github.com/scikit-rf/scikit-rf/issues/521

```

Comment:
I'm not sure if it is useful to refer to a GitHub issue in the docstring.
```suggestion
```

Reply:
Removed